coral-schema/src/test/java/com/linkedin/coral/schema/avro/TestUtils.java

```diff
@@ -106,6 +107,7 @@ private static void initializeTables() {
     executeCreateTableQuery("default", "basetimestamptypefield", baseTimestampTypeFieldSchema);
     executeCreateTableQuery("default", "basecomplexuniontype", baseComplexUnionTypeSchema);
     executeCreateTableQuery("default", "basenestedunion", baseNestedUnionSchema);
+    executeCreateTableQuery("default", "basecomplexunionforcelowercase", baseComplexUnionForceLowercase);
```

Comment:
I do not think we should encode the test case assertion logic in the name. The name should reflect the schema if you want it to be descriptive.